Source/ALSV4_CPP/Private/Character/ALSBaseCharacter.cpp

```diff
@@ -1279,6 +1279,8 @@ bool AALSBaseCharacter::MantleCheck(const FALSMantleTraceSettings& TraceSettings
 	return true;
 }
 
+//This function is called by "MantleTimeline" in the AALSBaseCharacter::BeginPlay, when default settings are set for the timeline.
+//Line 114 (Subject to change): TimelineUpdated.BindUFunction(this, FName(TEXT("MantleUpdate")))
```

Comment:
I would rather prefer not including that line. First line is enough as an explanation, and please put a space before the comment text to conform coding standards.